tests/test_broker.py

```
@@ -0,0 +1,249 @@
+                    "sddgov.broker._root_owned_directory_errors", return_value=[]
+                ), patch(
+                    "sddgov.broker.grp.getgrnam",
+                    return_value=SimpleNamespace(gr_gid=os.getgid()),
+                ), patch("sddgov.broker.os.chown"), patch(
+                    "sddgov.broker.os.chmod"
+                ), patch(
+                    "sddgov.broker.NonceLedger", return_value=fake_ledger
+                ), patch(
+                    "sddgov.broker.socket.socket",
+                    side_effect=lambda *_args, **_kwargs: SignalServer(
+                        signum, socket_path
```

Ruff (0.16.1)

[warning] Function definition does not bind loop variable `signum`

(B023)